packages/logger/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
```

Comment:
That option is being set here: https://github.com/nasa/cumulus/blob/CUMULUS-1883-migrate-logger-package/tsconfig.json#L6

We do want to publish that `.d.ts` file (and should with the current configuration). We don't want to commit it, though, since it's generated from the `.ts` files.

Reply:
I don't have strong feelings - but why not commit it? Even if it is generated, it will only change when changes are made in this package